### PR TITLE
Scenario Manager: add ports_per_vswitch in InternalTopologyConfig

### PR DESCRIPTION
**Pull request by cj-chung**
This PR make following change:
1) add `ports_per_vswitch` to `InternalTopologyConfig` protobuf
2) make the corresponding change in scenario manager's `entities.TopoConfig`
3) make the corresponding change in the constructTopology message.

### CODE REVIEW

**Review by phudtran — CHANGES_REQUESTED** [review]
Just some typos.

File: services/scenario-manager/handler/message.go [hunk 1]
```diff
@@ -37,6 +37,7 @@ func constructTopologyMessage(topo *entities.TopologyConfig, topoPb *topology_pb
 	conf.NumberOfVhosts = uint32(topo.NumberOfVhosts)
 	conf.NumberOfRacks = uint32(topo.NumberOfRacks)
 	conf.VhostPerRack = uint32(topo.VhostsPerRack)
+	conf.PortsPerVswtich = uint32(topo.PortsPerVSwitch)
```
[the remark this answers is not in the document]
Comment (cj-chung) [reply services/scenario-manager/handler/message.go:40]: fixed the typo

File: api/proto/v1/topology.proto [hunk 2]
```diff
@@ -77,13 +77,14 @@ message InternalTopologyConfiguration {
...
     uint32 number_of_racks = 9;
     uint32 vhost_per_rack = 10;
-    string data_plane_cidr = 11;
-    uint32 number_of_gateways = 12;
-    repeated string gateway_ips = 13;
-    repeated InternalTopologyImage images = 14;
-    repeated InternalVNodeInfo vnodes = 15;
-    repeated InternalVLinkInfo vlinks = 16;
-    InternalTopologyExtraInfo extra_info = 17;
+    uint32 ports_per_vswtich = 11;
```
[the remark this answers is not in the document]
Comment (cj-chung) [reply api/proto/v1/topology.proto:80]: fix the typo

**Review by phudtran — APPROVED** [review]
LGTM

**Pull request merged**